src/components/Buttons/IconButton/IconButton.tsx

```diff
@@ -13,6 +13,11 @@ type IconButtonProps = {
 	disabled?: boolean;
 };
 
+export const TEST_ID = {
+	CONTAINER: 'icon-button-container',
+	CONTENT: 'icon-button-image',
```

Comment:
This test_id is not used in testing suits.

I recommend to use the name: `BUTTON: 'icon-button'`, `ICON: 'icon-button-icon'`. `Container` does not make sense to me.

Reply:
@ericlowturing Thank you, need to work on my naming skills! Cheers. It's done.